build/gen-electron-builder-config.js

```diff
@@ -104,57 +105,67 @@ build['win'] = {
   files: [...files, PREBUILD_FILTERS.NOT_MAC, PREBUILD_FILTERS.NOT_LINUX],
 }
 
+// supported languages are on https://learn.microsoft.com/en-us/windows/apps/publish/publish-your-app/supported-languages?pivots=store-installer-msix
+const languages = [
+  'ar',
+  'bg',
+  'ca',
+  'cs',
+  // 'ckb', not supported by ms-store
+  'da',
+  'de',
+  'en',
+  'el',
+  // 'eo',  not supported by ms-store
+  'es',
+  'eu',
+  'fa',
+  'fi',
+  'fr',
+  'gl',
+  'hr',
+  'hu',
+  'id',
+  'it',
+  'ja-jp',
+  'km',
+  'ko',
+  'lt',
+  'nb',
+  'nl-nl',
+  'pl',
+  'pt',
+  'pt-BR',
+  'ro',
+  'ru',
+  // 'sc', not supported by ms-store
+  'sk',
+  'sq',
+  'sr',
+  'sv',
+  'ta',
+  'te',
+  'tr',
+  'uk',
+  'zh-cn',
+  'zh-tw',
+].map(code => code.toLowerCase())
+
+const unsupported_languages = languages.filter(
+  code => !isAppxSupportedLanguage(code)
+)
+if (unsupported_languages.length > 0) {
+  throw new Error(
+    'Unsupported appx languages:' + JSON.stringify(unsupported_languages)
+  )
+}
+
```

Comment:
Why not including all languages and then remove them if it's not supported?

Reply:
we could dynamically take them from languages.json, but there the language code might the exact one that ms supports/expect, so this gives us more control.